/robot/attractor:
  ros__parameters:
    basis:
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 1.0
    offset:
    - 0.0
    - 0.0
    - 0.0
    stiffness: 500.0
    damping: 50.0
    force_transform:
    - 1.0
    - 0.0
    - 0.0
    - 0.0
    - 1.0
    - 0.0
    - 0.0
    - 0.0
    - 1.0
    sample_interval_s: 0.0005
    publish_force: true

